Source/WebKit/UIProcess/API/Cocoa/_WKActivatedElementInfoInternal.h

```diff
@@ -46,5 +46,6 @@ namespace WebKit {
 @property (nonatomic, readonly) NSString *imageMIMEType;
 @property (nonatomic, readonly) WebCore::IntPoint _interactionLocation;
 @property (nonatomic, readonly) BOOL _isImage;
+@property (nonatomic) BOOL hasAlternateURLForImage;
```

Comment:
This could be a `readonly` property, and set in the initializer (`_initWithType:`) like the other properties, but I don't feel strongly about this, since the initializer is quite long already.

Reply:
I considered this as well, but ultimately decided against it since I noticed no other parameters in the constructor were `BOOL` types so I thought this would be more consistent albeit not `readonly`